repository/src/main/java/org/apache/atlas/repository/store/graph/v2/EntityGraphMapper.java

```diff
@@ -4514,5 +4514,27 @@ public void addHasLineage(Set<AtlasEdge> inputOutputEdges, boolean isRestoreEnti
         }
         RequestContext.get().endMetricRecord(metricRecorder);
     }
+    public void linkBusinessPolicy(String policyId, List<String> linkGuids) {
+        for (String guid : linkGuids) {
+            AtlasVertex ev = AtlasGraphUtilsV2.findByGuid(graph, guid);
+            if (ev != null) {
+                Set<String> existingValues = ev.getMultiValuedSetProperty("assetPolicyGUIDs", String.class);
```

Comment:
do we already have created these attributes on Asset typeDef `assetPolicyGUIDs` & `assetPoliciesCount`?
If not then these will never be written to Cassandra & eventually not sync to ES as well

Either add new attributes on Asset OR in case you will run tests on custom tenant you can update typeDefs directly using API & pause typeDef seeder cron